engine/src/cmdss.cpp

```diff
@@ -448,16 +448,22 @@ Parse_stat MCGo::parse(MCScriptPoint &sp)
 MCStack *MCGo::findstack(MCExecContext &ctxt, MCStringRef p_value, Chunk_term etype, MCCard *&cptr)
 {
     MCStack *sptr = nil;
+    sptr = MCInterfaceTryToEvalStackFromString(p_value);
+    if (sptr != nil)
+        return sptr;
+
     if (MCInterfaceStringCouldBeStack(p_value))
     {
-        sptr = MCInterfaceTryToEvalStackFromString(p_value);
-        if (sptr == nil)
-        {
+        //sptr = MCInterfaceTryToEvalStackFromString(p_value);
+        //if (sptr == nil)
+        //{
             if (MCresult->isclear())
                 ctxt . SetTheResultToCString("can't build stack from string");
-        }
+        //}
         return sptr;
     }
+    else
+        MCresult->clear();
     
 	if (etype == CT_STACK)
         return NULL;
```

Comment:
I didn't know we had a preference either @mwieder ... actually I thought the preference was `nil`.. Anyway, I will also use `nullptr` from now on. Actually having looked again I see it is mentioned in initialising local variables here https://github.com/livecode/livecode/blob/develop/docs/development/C%2B%2B-style.md